libraries/AP_OpticalFlow/AP_OpticalFlow_PX4Flow.h

```diff
@@ -3,6 +3,8 @@
 #include "OpticalFlow.h"
 #include <AP_HAL/utility/OwnPtr.h>
 
+#define AP_OPTICALFLOW_PX4FLOW_INIT_RETRIES 10  // attempt to initialise the sensor up to 10 times at startup
```

Comment:
Thanks for the review.  I've moved the define to the .cpp.

Re a new "active" method, I'm not too keen on adding new methods that could add confusion.  The difference could be a bit subtle vs enabled() and healthy() so I think I prefer that we just keep logging even though the data it will record will be all zeros.  Same for the other things tied to "enabled()".